resources/building-mobile-extensions/event-listeners.md

```diff
@@ -166,7 +117,66 @@ public class MyExtension extends Extension {
 }
 ```
 
-## What can you do in your event handler?
+{% endtab %}
+
+{% tab title="Objective-C" %}
+
+### **iOS**
+
+1. In Xcode, create a new file from the `Cocoa Touch Class` template and save it in your project.
+2. Name your class `MyExtensionListener`, and it should be a subclass to the `ACPExtensionListener` class.
+
+   The `MyExtensionListener.m` file will contain your extension interface declaration and will import `ACPExtensionListener.h`. In the example below, the methods that are available for overriding are also displayed:
```

Comment:
please change "will contain" to "contains" and change "will import" to "imports"